### ISSUE TITLE
Add `--wait` functionality to `acorn run`

### ISSUE THREAD

**Issue by cjellick** [acorn-io/runtime#548]
When we run acorn apps, we find ourselves doing:
```
watch acorn ps
```
to wait until the app becomes healthy. Want to add  `--wait` flag so that the run command blocks until the cotainers are all reporting healthing and (maybe?) the endpoint is out of pending.

**Comment by iwilltry42** [comment]
`acorn run --wait` already exists and is enabled by default.
However, it may not wait for absolutely everything to be 100% functional.
Here's an example:

```shell
acorn run --wait .local 
[+] Building 2.4s (5/5) FINISHED                                                                                                                                                                                    
 # ...
 => => pushing manifest for 127.0.0.1:5000/acorn/acorn:latest@sha256:ce248997aac1268a0a70a7f275d3f64f39b2116311aa32452323362078fe824c                                                                          0.0s
autumn-meadow
STATUS: ENDPOINTS[] HEALTHY[] UPTODATE[] 
STATUS: ENDPOINTS[] HEALTHY[0] UPTODATE[0] pending
STATUS: ENDPOINTS[] HEALTHY[0/1] UPTODATE[1] [containers: autumn-meadow is not ready; my-webapp ContainerCreating]
STATUS: ENDPOINTS[http://my-webapp.autumn-meadow.local.on-acorn.io => my-webapp:80] HEALTHY[0/1] UPTODATE[1] [containers: autumn-meadow is not ready; my-webapp ContainerCreating]
```

So the output doesn't end with a success and if watching the pod status at the side, one can see that the pod is still in `ContainerCreating` state as the acorn command returns. That may even be correct as the Acorn in this example doesn't specify any probes.

Is the current behavior good enough or should we define more strict rules on when an Acorn App is considered ready?

**Comment by cjellick** [comment]
@ibuildthecloud what's up here? What status are you waiting on with the `--wait` behavior?

**Comment by ibuildthecloud** [comment]
@cjellick @iwilltry42 It just waits for `app.Status.Ready` to be true.  If that is true and the app is not in fact ready then the controller that updates the status should be modified.  One issue with the Ready check is that it won't consider "Uptodate"  so it is possible if you are updating an app it will be considered ready when it has updated.  We should probably add another field status.UpToDate to indicates that the spec and status match.

I believe there is probably another issue in that when spec.Stop = true we should be setting ready to false.
